extensions/commands/maintenance/cmd_clean_old_revisions.py

```diff
@@ -0,0 +1,53 @@
+from conan.api.conan_api import ConanAPI
+from conan.api.output import ConanOutput, Color
+from conan.cli.command import OnceArgument, conan_command
+from conans.client.userio import UserInput
+
+
+recipe_color = Color.BRIGHT_BLUE
+removed_color = Color.BRIGHT_YELLOW
+
+
+@conan_command(group="Extensions")
+def clean_old_revisions(conan_api: ConanAPI, parser, *args):
+    """
+    Deletes (from local cache or remotes) all recipe and package revisions but
+    the latest package revision from the latest recipe revision.
+    """
+    parser.add_argument('-r', '--remote', action=OnceArgument,
+                        help='Will remove from the specified remote')
+    parser.add_argument('--force', default=False, action='store_true',
+                        help='Remove without requesting a confirmation')
```

Comment:
Confirming should be `-c` like `conan remove`

```
  -c, --confirm         Remove without requesting a confirmation
```

```suggestion
    parser.add_argument('-c', '--confirm', default=False, action='store_true',
                        help='Remove without requesting a confirmation')
```